syntax = "proto2";

package opentxs.proto;
option optimize_for = LITE_RUNTIME;

import public "Enums.proto";
import public "AsymmetricKey.proto";

message KeyCredential {
    optional uint32 version = 1;
    optional KeyMode mode = 2;
    repeated AsymmetricKey key = 3;
}
